x/msgfees/client/cli/cli_test.go

```diff
@@ -70,72 +70,137 @@ func (s *IntegrationTestSuite) TearDownSuite() {
 
 func (s *IntegrationTestSuite) TestMsgFeesTxGovProposals() {
 	testCases := []struct {
-		name         string
-		args         []string
-		expectErr    bool
-		expectErrMsg string
-		expectedCode uint32
+		name          string
+		typeArg       string
+		title         string
+		description   string
+		deposit       string
+		msgType       string
+		additionalFee string
+		recipient     string
+		bips          string
+		expectErr     bool
```

Comment:
I know this was already there, but it should be removed. If we're expecting an error, we should be checking the content of the returned error against an expected value (or at least checking that it contains certain pieces). So you can use `len(expectErrMsg) != 0` in place of this `expectErr bool`.

Reply:
Yep, I was cleaning these up in this PR.  Missed these... 👍